iml-agent/src/network_interface_stats.rs

```diff
@@ -5,7 +5,9 @@ use combine::{
     stream::Stream,
     token, Parser,
 };
-use std::collections::HashMap;
+use std::{collections::HashMap, convert::TryFrom, num::ParseIntError};
+
+type MaybeStat = Result<u64, ParseIntError>;
```

Comment:
nit: `MaybeStat` makes me think of an option. `StatResult` would probably be a good name.

Reply:
Sure, I'll update.